robottelo/helpers.py

```diff
@@ -143,3 +143,53 @@ def read_data_file(filename):
     absolute_file_path = get_data_file(filename)
     with open(absolute_file_path, 'r') as file_contents:
         return file_contents.read()
+
+
+def install_katello_ca(hostname=None):
+        """Downloads and installs katello-ca rpm
+
+        :param str hostname: Hostname or IP address of the remote host. If
+        ``None`` the hostname will be get from ``main.server.hostname`` config.
+        :return: None.
+        :raises AssertionError: If katello-ca wasn't installed.
+        """
+        ssh.command(
+            u'rpm -Uvh {0}'.format(settings.server.get_cert_rpm_url()),
+            hostname
+        )
+        # Not checking the return_code here, as rpm could be installed before
+        # and installation may fail
+        result = ssh.command(
+            u'rpm -q katello-ca-consumer-{0}'
+            .format(settings.server.hostname),
+            hostname
+        )
+        # Checking the return_code here to verify katello-ca rpm is actually
+        # present in the system
+        if result.return_code != 0:
+            raise AssertionError('Failed to install the katello-ca rpm')
+
+
+def remove_katello_ca(hostname=None):
+        """Removes katello-ca rpm
+
+        :param str hostname: Hostname or IP address of the remote host. If
+        ``None`` the hostname will be get from ``main.server.hostname`` config.
+        :return: None.
+        :raises AssertionError: If katello-ca wasn't removed.
+        """
+        ssh.command(
```

Comment:
I'm 99.99% sure that it doesn't hurt to remove `katello-ca`. The 'side effect' is that if you wanted to yum install anything in the Satellite system itself, you'd get a big fat zero since the Satellite would be configured against itself, unless the Satellite has been registered against an org with proper content.